javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (peopleObj) => {
```

Comment:
A recommendation, `peopleObj` should be pluralized to `peopleObj` since `getNames` will take in an array of people objects.